doc/cascadia/profiles.schema.json

```diff
@@ -0,0 +1,449 @@
+{
+  "$id": "https://github.com/microsoft/terminal/terminal.profile.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Microsoft Terminal Profile Settings",
+  "definitions": {
+    "Color": {
+      "default": "#",
+      "pattern": "^#[A-Fa-f0-9]{6}$",
+      "type": "string"
+    },
+    "ProfileKey": {
+      "default": "{}",
```

Comment:
I'm not totally sure this is true, but I'm not sure what a better way to put this would be